test/blackbox-tests/test-cases/watching/fs-memo.t

```diff
@@ -287,12 +287,12 @@ Same problem for files.
   Error: file-1: Permission denied
   -> required by _build/default/file-1
   -> required by _build/default/result
-  -> required by alias default
+  -> required by alias default in dune:1
   Had 1 error, waiting for filesystem changes...
   Error: file-1: Permission denied
   -> required by _build/default/file-1
   -> required by _build/default/result
-  -> required by alias default
+  -> required by alias default in dune:1
```

Comment:
I believe this is due to the `loc` change (where it wasn't passed but is now). I suggest splitting this out into a separate PR tbh…

Reply:
I've split it out in #11705, so when rebasing this will be omitted :)

Reply:
Thanks!

Reply:
Done in [50fe3d3](https://github.com/ocaml/dune/pull/11609/commits/50fe3d388bada3d7c9272a23b227260000733e95)